infrastructure/database.tf

```diff
@@ -41,3 +42,48 @@ resource "azurerm_key_vault_secret" "POSTGRES-HOST" {
   value        = module.postgresql.fqdn
   key_vault_id = module.civil_sdt_key_vault.key_vault_id
 }
+
+
+module "postgresql-v15" {
+  source = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+
+  providers = {
+    azurerm.postgres_network = azurerm.private_endpoint
+  }
+
+  admin_user_object_id = var.jenkins_AAD_objectId
+  business_area        = "cft"
+  common_tags          = local.tags
+  component            = var.component-V15
```

Comment:
The component variable can just be set to `var.component`.

The component input variable is only ever used to set the [default_name local variable](https://github.com/hmcts/terraform-module-postgresql-flexible/blob/master/pgsql-flexible-server.tf#L2) and the default_name local variable is only ever used to set the [name local variable](https://github.com/hmcts/terraform-module-postgresql-flexible/blob/master/pgsql-flexible-server.tf#L3) if the name input variable isn't set.  The name input variable is being set, so the value of this is basically ignored.

Reply:
removed it now